tests/unit/test_template.py

```
@@ -60,91 +62,139 @@ def test_external_workflow(self):
+				'test2': {}
 			}
 		}
-
-	def test_extract_tasks_and_supported_opts_task(self):
-		# Test different configurations
-		from secator.tasks import nuclei
+		self.template_dir = CONFIG.dirs.templates
+		self.custom_task_path = self.template_dir / 'test.py'
+		self.custom_workflow_path_1 = self.template_dir / 'test.yml'
+		self.custom_workflow_path_2 = self.template_dir / 'test2.yml'
+		with open(self.custom_task_path, 'w') as f:
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)